build.mill

```diff
@@ -227,6 +227,13 @@ object ujson extends Module{
       def mvnDeps = Seq(mvn"io.circe::circe-parser:$circeVersion")
     }
 
+    object js extends Cross[CirceJsModule](scalaVersions)
+    trait CirceJsModule extends CommonJsModule{
+      def moduleDeps = Seq(ujson.js())
+      val circeVersion = "0.14.14"
+      def mvnDeps = Seq(mvn"io.circe::circe-parser:$circeVersion")
```

Comment:
Can you extract these in a trait as we do in `CommonCoreModule` and then extend it in jvm, js and native?

Reply:
Sure thing, done